website/content/docs/platform/k8s/vso/api-reference.mdx

```diff
@@ -8,18 +8,18 @@ description: >-
 @include 'vso-beta-note.mdx'
 
 <!--
-  generated with crd-ref-docs --source-path api/v1alpha1 --config docs/config.yaml --renderer=markdown in the vault-secrets-operator repo.
-  commit SHA=f3752fbce7be3bb9e73615a10d2ddeb58e2a8cb6
+  generated with crd-ref-docs --source-path api/v1beta1 --config docs/config.yaml --renderer=markdown in the vault-secrets-operator repo.
```

Comment:
Now that we have the new make target I think we can probably just say `copied from vault-secrets-operator/docs/api/api-reference.md` or something like that, instead of the generated stuff. Definitely keep the commit SHA.